src/unix/bsd/apple/mod.rs

```diff
@@ -3528,6 +3560,22 @@ extern "C" {
         newfd: ::c_int,
     ) -> ::c_int;
     pub fn uname(buf: *mut ::utsname) -> ::c_int;
+
+    pub fn connectx(
+        socket: ::c_int,
+        endpoints: *const sa_endpoints_t,
+        associd: sae_associd_t,
+        flags: ::c_uint,
+        iov: *const ::iovec,
+        iovcnt: ::c_uint,
+        len: *mut ::size_t,
+        connid: *mut sae_connid_t,
+    ) -> ::c_int;
+    pub fn disconnectx(
+        socket: ::c_int,
+        associd: sae_connid_t,
```

Comment:
It isn't `sae_associd_t`?
```suggestion
        associd: sae_associd_t,
```

Reply:
Ah! Yes!